ReactNativeClient/lib/models/Note.js

```diff
@@ -85,13 +85,19 @@ class Note extends BaseItem {
 		if (body && body.length) {
 			const lines = body.trim().split('\n');
 			let output = lines[0].trim();
-			// Remove the first #, *, etc.
-			while (output.length) {
-				const c = output[0];
-				if (['#', ' ', '\n', '\t', '*', '`', '-'].indexOf(c) >= 0) {
-					output = output.substr(1);
-				} else {
-					break;
+			if (output.match(/\[.+\]\(.+\)/)) {
+				const start = output.indexOf('[') + 1;
+				const stop = output.indexOf(']') - 1;
```

Comment:
Maybe I'm nitpciking here but the 2nd parameter of substr is not the end index but length.
In this case the result would always be the same thanks to the trim call above.

Reply:
Ah, thanks for pointing that out! Using the wrong function